src/main/kotlin/com/github/pool_party/pull_party_bot/commands/PingCommandHandlers.kt

```diff
@@ -8,16 +8,14 @@ import com.github.pool_party.pull_party_bot.database.createCommandTransaction
 import com.github.pool_party.pull_party_bot.database.deleteCommandTransaction
 import com.github.pool_party.pull_party_bot.database.listCommandTransaction
 import com.github.pool_party.pull_party_bot.database.partyCommandTransaction
-import com.github.pool_party.pull_party_bot.database.rudeCheckTransaction
 import com.github.pool_party.pull_party_bot.database.rudeCommandTransaction
-import java.lang.StringBuilder
 
 fun Bot.initPingCommandHandlers() {
     onNoArgumentsCommand("/start", ::handleStart)
     onNoArgumentsCommand("/help", ::handleHelp)
     onNoArgumentsCommand("/list", ::handleList)
 
-    onCommand("/party", ::handleParty)
+    onCommand("/party", ::handleExplicitParty)
```

Comment:
Why is that? My logic is as follows: using command is explicit, because u explicitly mention that u r about to tag people; using `@` tag mention in the message is implicit, because it is embedded in the normal chatting messages.